README.md

```diff
@@ -1,4 +1,4 @@
-[link-amo]: https://addons.mozilla.org/en-US/firefox/addon/cookie-autodelete/
+[link-amo]: https://addons.mozilla.org/firefox/addon/cookie-autodelete/
```

Comment:
Doesn't appear this change would make any difference - for me the revised link would still redirect to include `en-US`, but if AMO is automatically adding it in then removing it wouldn't hurt to allow for auto localization of webpage to their own language.

Reply:
> Doesn't appear this change would make any difference - for me the revised link would still redirect to include `en-US`, but if AMO is automatically adding it in then removing it wouldn't hurt to allow for auto localization of webpage to their own language.

Yes, AMO auto-localizes for the user's locale.  Thus it's best practice to not force a locale.

Reply:
For Firefox 123+ at Android could be shorter:

https://addons.mozilla.org/firefox/addon/cookie-autodelete/

Firefox less 123 may need use link for PC/macOS with "/firefox".